tests/report/common/tests.cpp

```diff
@@ -220,19 +223,43 @@ TEST_FCN void TestLocalReport(void* args_)
         oe_result_t expected_result = OE_OK;
         report_size = 1024 * 1024;
         OE_TEST(
-            GetReport(0, NULL, 0, NULL, 0, report_buffer, &report_size) ==
+            GetReport_v1(0, NULL, 0, NULL, 0, report_buffer, &report_size) ==
+            OE_OK);
+
+        if (expected_result == OE_OK)
+        {
+            ValidateReport(
+                report_buffer, report_size, false, zeros, OE_REPORT_DATA_SIZE);
+        }
+
+        OE_TEST(
+            GetReport_v2(
+                0, NULL, 0, NULL, 0, &report_buffer_ptr, &report_ptr_size) ==
+            OE_OK);
+
+        if (expected_result == OE_OK)
+        {
+            ValidateReport(
+                report_buffer, report_size, false, zeros, OE_REPORT_DATA_SIZE);
```

Comment:
Validate with `report_buffer_ptr` and `report_ptr_size` instead?

Reply:
good catch.